### PR TITLE
config: make some scheduling settings more aggressive

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Some settings may limit the speed of scheduling in some cases.

### What is changed and how it works?

This PR makes some scheduling settings more aggressive.

### Check List

Tests

- Unit test

Code changes

- Has configuration change

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):

### Release note

```release-note
Please add a release note.

If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

**Review by disksing — APPROVED** [review]

**Review by HunDunDM — COMMENTED** [review]
How about changing the `defaultLeaderScheduleLimit` to `16`? It needs to be greater than `defaultMergeScheduleLimit`.

**Comment by HunDunDM** [comment]
#3820 has been merged, does this PR need to be modified?

File: server/config/config.go [hunk 1]
```diff
@@ -748,12 +748,12 @@ func (c *ScheduleConfig) Clone() *ScheduleConfig {
 
 const (
 	defaultMaxReplicas               = 3
-	defaultMaxSnapshotCount          = 3
-	defaultMaxPendingPeerCount       = 16
+	defaultMaxSnapshotCount          = 64
+	defaultMaxPendingPeerCount       = 64
```
Comment (nolouch) [inline server/config/config.go:752]: Maybe make this bigger?

**Review by nolouch — APPROVED** [review]

**Comment by rleungx** [comment]
Closes #https://github.com/tikv/pd/pull/3685

**Pull request merged**